### ISSUE TITLE
Error parsing template admin_tools/menu/dummy.html: Invalid template name in 'extends' tag: ''

### ISSUE THREAD

**Issue by Ash-Crow** [django-admin-tools/django-admin-tools#120]
I have the following error from django-admin-tools when using django-compressor: 

```
$ python manage.py compress --force
Compressing... Error parsing template admin_tools/menu/dummy.html: Invalid template name in 'extends' tag: ''. Got this from the 'template' variable.
Error parsing template admin_tools/dashboard/dummy.html: Invalid template name in 'extends' tag: ''. Got this from the 'template' variable.
done
Compressed 22 block(s) from 202 template(s) for 1 context(s).
```

Versions:
- Python 3.10.1
- Django==3.2.12
- django-admin-tools==0.9.2

**Comment by gauravgupta369** [comment]
Can you use **{% include template %}** instead {% extends template %} in dymmy.html?

**Comment by Ash-Crow** [comment]
I think this PR #127 would solve the issue.

**Comment by izimobil** [comment]
Fixed with new 0.9.3 release, thanks.

**Comment by gauravgupta369** [comment]
Thank You @izimobil Its working fine with the 0.9.3 release.
